docs/index.rst

```diff
@@ -25,6 +25,7 @@
    Platform Support <platform-support.rst>
    Getting Started with Kubernetes <getting-started-with-kubernetes.rst>
    Getting Started with Red Hat OpenShift <getting-started-with-openshift.rst>
+   NVIDIA Network Operator Government Ready <install-network-operator-gov-ready.rst>
```

Comment:
Do we need `NVIDIA Network Operator` in the tree link?
Maybe stick with same wording as `Getting Started with`